packages/cli/src/active-workflow-manager.ts

```diff
@@ -90,7 +90,6 @@ export class ActiveWorkflowManager {
 		await this.addActiveWorkflows('init');
 
 		await this.externalHooks.run('activeWorkflows.initialized');
-		await this.webhookService.populateCache();
```

Comment:
We are [already populating the cache](https://github.com/n8n-io/n8n/blob/4577ce08468c680fa8e9b8ed2ba6acb17c80bd36/packages/cli/src/active-workflow-manager.ts#L222) immediately after adding the webhooks for every activated workflow.

Reply:
Not related to this PR, but noticed this: Combined with #13191, we actually now query all webhooks multiple time and add them to the cache multiple times. Shouldn't we only add that one specific WFs webhooks in the `addWebhooks`?